nested-data-and-loops/nested-lists-and-dictionaries.md

```diff
@@ -0,0 +1,280 @@
+sarahs_list[3][2] = '🤠'
+sarahs_list[4][1] = '🤠'
+
+print(sarahs_list)
+```
+
+## Dictionaries Containing Lists
+
+Dictionaries can contain lists.
+
+```python
+{
+    'oranges' : ['a', 'b', 'c', 'd'],
+    'apples' : ['e', 'f']
+}
+```
+
+In this example:
+- The outer dictionary has two key-value pairs: `'oranges'` and `'apples'`
+- One inner list has four elements
+- The other inner list has two elements
+
+### Accessing One Element
+
+To access one element, we chain the syntax to get a value from a dictionary, and then index a list.
+
+```python
+example_dict[ key_to_find_inner_list ][ index_to_find_element ]
+```
+
+Callout: Again, we practice reading this code from left-to-right. First, we evaluate the expression `example_dict[ key_to_find_inner_list ]`. Then, moving to the right, we index that list with `index_to_find_element`.
+
+Observe these examples:
+```python
+sarahs_dict = {
+    'oranges' : ['a', 'b', 'c', 'd'],
+    'apples' : ['e', 'f']
+}
+
+b = sarahs_dict['oranges'][1]
+d = sarahs_dict['oranges'][3]
+e = sarahs_dict['apples'][0]
+```
+
+### Modifying An Element
+
+This is example syntax for modifying one element in a nested list.
+
+```python
+example_list[ index_to_find_inner_list ][ index_to_find_element ] = 'This is my new value!'
+```
+
+Observe these examples:
+```python
+sarahs_dict = {
+    'oranges': ['a', 'b', 'c', 'd'],
+    'apples': ['e', 'f']
+}
+
+sarahs_dict['oranges'][1] = '🍊'
+sarahs_dict['oranges'][3] = '🍊'
+sarahs_dict['apples'][0] = '🍎'
+```
+
+## Lists Containing Dictionaries
+
+Lists can contain dictionaries.
+
+```python
+[
+    {
+        'orange': '🍊',
+        'apple': '🍎'
+    },
+    {
+        'bird': '🐦',
+        'dog': '🐶'
+    }
+]
+```
+
+In this example:
+- The outer list has two items
+- The first inner dictionary has two key-value pairs
+- The second inner dictionary has two key-value pairs
+
+### Accessing One Value
+
+To access one value from a nested dictionary, we chain the syntax to get a value from a dictionary.
+
+```python
+example_list[ index_to_find_item ][ key_to_access_value ]
+```
+
+Callout: Again, we practice reading this code from left-to-right. First, we evaluate the expression `example_dict[ key_to_find_inner_list ]`. Then, moving to the right, we index that list with `index_to_find_element`.
+
+Observe these examples:
+```python
+sarahs_list = [
+    {
+        'orange': '🍊',
+        'apple': '🍎'
+    }
+]
+
+orange_emoji = sarahs_list[0]['orange']
+apple_emoji = sarahs_list[0]['apple']
+```
+
+### Modifying A Value
+
+Modifying the contents of a nested dictionary follows the syntax rules we've seen before. We want to re-assign a value, so we use the assignment operator.
+
+Observe these examples:
+```python
+sarahs_list = [
+    {
+        'orange': '🍊',
+        'apple': '🍎'
+    }
+]
+
+sarahs_list[0]['orange'] = 'A fruit that is round and orange'
+sarahs_list[0]['apple'] = 'A fruit with a lot of varieties'
+```
+
+## Dictionaries Containing Dictionaries
```

Comment:
This Dictionary of Dictionaries section feels like it's running out of steam.  I know this is the fourth time essentially the same material is stated, but consider expanding the verbiage a bit to give it the same consideration, or use explanations that recognize the similarity of this material with the previously three sections with the acknowledged intent to keep this final section brief.